distribution/ltp/lite/runtest.sh

```diff
@@ -191,7 +191,7 @@ fi
 # report patch errors from ltp/include
 grep -i -e "FAIL" -e "ERROR" patchinc.log > /dev/null 2>&1
 if [ $? -eq 0 ]; then
-    rhts-abort -t recipe
+	report_result "ltp-include-patch-errors" "WARN"
```

Comment:
The warnings should only be reported if the tests really fail, not if they fail to install / be patched - that's a test problem and not a kernel one; please keep the previous version